### DISCUSSION TITLE
IR send add no modulation option

### DISCUSSION THREAD

**Discussion by farcouet** [arendst/Tasmota#18453] [Ideas]
Hello,
I would suggest a small improvement to the code to enable sending IR commands without any carrier frenquency.

The idea for this is to hardwire directly the GPIO to the device to control directly at the output of the IR receiver module. The reason is that way it's easy to be sure that the command has been received by the  AC unit without any risk of interference. This is how I am currently doing without Tasmota and now Tasmota is so powerful to decode and encode Gree for example that I would really like to move to Tasmota. 

I am also receiving the IR signal that the AC is receiving (when someone is controlling with the IR remote) the same way. But for receiving no modification to the code is needed as the signal is coming from the IR receiver module of the AC so it can be done directly.

Thanks!
Fred

**Comment by barbudor** [comment]
This is already available as a built-time option
Set the below in your user_config_override.h.
```
  #define IR_SEND_USE_MODULATION    false
```

**Reply by farcouet** [reply]
Ok thanks I did not looked in the code. I'll recompile using that option! 
Thanks a lot!
